docs/deployment/image.md

```diff
@@ -11,7 +11,7 @@ docker build -t demo-inveniordm /path/to/your/instance --build-arg include_asset
 
 **Why is the `include_assets` flag needed**
 
-When running InvenioRDM in local (`invenio-cli run`), you are running the uWSGI server on your own machine where you have built the statics. On the other hand, when running `invenio-cli containerize` you are building the docker image and then running it along the other containers. There, `invenio-cli` takes care of building the statics in the mounted volume for you. However, when building the image on your own (e.g. deploying somewhere else), you need to take care of the statics yourself.
+When running InvenioRDM in local (`invenio-cli run`), you are running the uWSGI server on your own machine where you have built the statics. On the other hand, when running `invenio-cli containers start` you are building the docker image and then running it along the other containers. There, `invenio-cli` takes care of building the statics in the mounted volume for you. However, when building the image on your own (e.g. deploying somewhere else), you need to take care of the statics yourself.
```

Comment:
```suggestion
When running InvenioRDM locally (`invenio-cli run`), you are running the uWSGI server on your own machine where you have built the statics. On the other hand, when running `invenio-cli containers start` you are building the docker image and then running it along the other containers. There, `invenio-cli` takes care of building the statics in the mounted volume for you. However, when building the image on your own (e.g. deploying somewhere else), you need to take care of the statics yourself.
```

This is the difficulty of using options `--development/--production` and running `containers` versus `run`... all of these have asterisks next to them... boils down to having super clear cut use cases (the using invenio-cli in production is not clear cut yet- personally not sure if that's realistic without lot more development on it).